shared/chat/conversation-list/choose-conversation.js

```diff
@@ -22,27 +23,31 @@ class ChooseConversation extends React.Component<Props & Kb.OverlayParentProps,
   _toggleOpen = () => {
     this.setState(prevState => ({expanded: !prevState.expanded}))
   }
-  _onSelect = () => {
-    this.setState({expanded: false})
-  }
+  _onDone = () => this.setState({expanded: false})
   render() {
     return (
       <>
         <Kb.DropdownButton
-          disabled={this.props.disabled}
-          selected={<Kb.Text type="BodySemibold">{this.props.dropdownButtonDefaultText}</Kb.Text>}
+          disabled={false}
```

Comment:
🔥

Reply:
You mean we should keep this prop? This is only used in the attachment popup. I think it's always enabled.

Reply:
no i mean you can just delete it, it defaults to false right

Reply:
Ah that's right!